.buildkite/scripts/steps/fleet.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+preinstalled_packages_filename="testing/integration/testdata/preinstalled_packages.json"
+
+# preinstall_fleet_packages installs EPM packages needed for integration tests into Fleet.
+function preinstall_fleet_packages() {
+  echo "Preinstalling the following EPM packages in Fleet:"
+  cat "$preinstalled_packages_filename"
+
+  install_fleet_packages "$preinstalled_packages_filename"
+  retcode=$?
+
+  return $retcode
+}
+
+# install_fleet_packages installs the EPM packages defined in the given file into Fleet. The file
+# must define the packages as the request body of the Bulk install packages Fleet API:
+# https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-bulk-install-packages
+function install_fleet_packages() {
+  local install_api_request_file=$1
```

Comment:
Do you think we should avoid any 'verbose output' by forcing `set +x`?

```suggestion
  set +x
  local install_api_request_file=$1
```

Reply:
My preference would be to keep the verbose output for now, while this change is still somewhat new, in case it becomes helpful for debugging any issues.  Once things start running smoothly, we can add `set +x`.